src/loki/polygon_search.cc

```diff
@@ -32,13 +32,23 @@ static const auto Haversine = [] {
   return bg::strategy::distance::haversine<float>(vm::kRadEarthMeters);
 };
 
+void correct_ring(ring_bg_t& ring) {
```

Comment:
reminds me what I always wanted to ask: what's the convention in this repo for function names? pascalcase for public API, snake for internal? couldn't really make sense of it yet but was also never complained about :)

Reply:
once upon a time we made the huge mistake of using googles coding guidelines. they basically look like a java developer figured c++ was close enough to use the same style. i assume their motivation was minimal friction between moving around projects for developers inside of google.

i personally hated this style and whenever someone else hadn't already had tons of established google-style code i used my preferred STL-like underscores and lower case. basically a lot of it is my fault and i've caught a lot of flack for it over the years :stuck_out_tongue_winking_eye: